docs/_markbind/layouts/devGuide.md

```diff
@@ -3,7 +3,7 @@
 <div id="flex-body">
   <nav id="site-nav">
     <div class="site-nav-top">
-      <div class="fw-bold mb-2" style="font-size: 1.25rem;">Developer Guide</div>
+      <div class="fw-bold mb-2" style="font-size: 1.4rem;">Developer Guide</div>
```

Comment:
Same as previously commented, if we don't change this, does it still look ok?
By the way I think it's fine if it becomes a breaking change, just that we might need to delay this to the end of the semester or some time later.

Reply:
Yup it does! Just a smaller difference in font size between the `site-nav-top` and 0th tier of the `site-nav`. I would prefer this PR to not be a breaking change for grading purposes. Perhaps I could open a new PR in the future for these breaking changes instead?